frontend/src/components/forms/GeneralInfo.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 
 const GeneralData = () => {
+  const [name, setName] = React.useState('');
```

Comment:
What if you have a thousand fields? Will you create a thousand of useStates?

I think in this case we can create a state with an object. For example:

const [data, setData] = useState({});

And when you use it you can do it this way

value={data.name || ''}
onChange={(e) => setData({ ...data, name: e.target.value })}

Reply:
Solved, thanks for this new approach proposal, for this kind of feedback is important the code reviews.